app/services/site_setting/policy/settings_are_not_deprecated.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class SiteSetting::Policy::SettingsAreNotDeprecated < Service::PolicyBase
+  delegate :options, :params, to: :context
+
+  def call
+    params.settings.keys.each do |id|
+      SiteSettings::DeprecatedSettings::SETTINGS.find do |old_name, new_name, override, _|
```

Comment:
What do you think about using select and getting array of old_names and report back all of them?

That would be consistent with other policies reporting back all "problems" - https://github.com/discourse/discourse/blob/main/app/services/site_setting/policy/settings_are_configurable.rb